CONTRIBUTING.md

```diff
@@ -19,6 +19,9 @@ Thank you!
     ```
     bundle exec rake spec
     ```
+    
+    _NOTE: You may need to install the bundle with the `test` group first by running `bundle install --with test`. Otherwise, you will get `can't find executable rake for gem rake` error._
```

Comment:
I think it makes sense to have the installation step as its own bullet point in the contributing guide as hopefully then contributors won't see the error at all. So maybe something like this as the first bullet in this "Polish" section would be better:

```markdown
* Install the test dependencies
    ```
    bundle install --with test sidekiq --binstubs
    ```
```

Reply:
Also, in the README it says the command to run the tests is `bundle exec rake spec` whereas the TESTING file says `bundle exec rake`. Which one do you prefer?

Reply:
I think following the testing guide makes sense, so `bundle exec rake` — `spec` is the default so there's no real difference 🙂

TESTING.md isn't mentioned in the readme, but I think that's OK after this PR. If all the required steps for the unit tests are in CONTRIBUTING (which is mentioned in the readme) then the rest of the testing guide is irrelevant for outside contributors as the end to end tests can only be run with private credentials